Blockstack/Classes/BlockstackAuthentication.swift

```diff
@@ -10,20 +10,15 @@ import SafariServices
 import JavaScriptCore
 import secp256k1
 
-struct BlockstackConstants {
-    static let DefaultCoreAPIURL = "https://core.blockstack.org"
-    static let BrowserWebAppURL = "https://browser.blockstack.org"
-    static let BrowserWebAppAuthEndpoint = "https://browser.blockstack.org/auth"
-    static let AuthProtocolVersion = "1.1.0"
-}
-
-open class BlockstackInstance {
-    open var coreAPIURL = BlockstackConstants.DefaultCoreAPIURL
+open class BlockstackAuthentication {
```

Comment:
Repurposing `BlockstackInstance` to `BlockstackAuthentication` which would help in keeping SRP in the future and draw a line between different Blockstack modules